eng/versioning/version_client.txt

```diff
@@ -101,7 +101,7 @@ com.azure:azure-digitaltwins-core;1.3.0;1.4.0-beta.1
 com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-identity;1.5.4;1.6.0-beta.1
 com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-iot-deviceupdate;1.0.0-beta.3;1.0.0-beta.4
+com.azure:azure-iot-deviceupdate;1.0.0-beta.4;1.0.0
```

Comment:
It should be `com.azure:azure-iot-deviceupdate;1.0.0-beta.3;1.0.0`

First version is one of the last released, second version is the one that to be released next.

Reply:
Thank you.